scss/main.scss

```diff
@@ -102,3 +102,5 @@ nav {
     }
   }
 }
+
+@import 'responsive';
```

Comment:
It should be able to overwrite the CSS' that are written in the `main.scss` (preferably without using `!important`). If possible, we could move the actual style declarations from `main.scss` to an external file, so the `main.scss` would just be the file that only handles the imports.